dtfs-central-api/src/repositories/durable-functions-repo.ts

```diff
@@ -1,21 +1,27 @@
-import { DeleteResult } from 'mongodb';
-import { DURABLE_FUNCTIONS_LOG, MONGO_DB_COLLECTIONS } from '@ukef/dtfs2-common';
+import { AuditDetails, DURABLE_FUNCTIONS_LOG, MONGO_DB_COLLECTIONS } from '@ukef/dtfs2-common';
+import { deleteMany } from '@ukef/dtfs2-common/change-stream';
 import db from '../drivers/db-client';
 
-const getDurableFunctionsLogCollection = async () => db.getCollection(MONGO_DB_COLLECTIONS.DURABLE_FUNCTIONS_LOG);
-
-export const deleteAllDurableFunctionLogs = async (): Promise<DeleteResult> => {
-  const durableFunctionLogsCollection = await getDurableFunctionsLogCollection();
-  return durableFunctionLogsCollection.deleteMany({});
+export const deleteAllDurableFunctionLogs = async (auditDetails: AuditDetails): Promise<{ acknowledged: boolean }> => {
+  return await deleteMany({
```

Comment:
```suggestion
  return deleteMany({
```

Reply:
Not a big issue at all but we don't need await here - when we await the result before returning it, the function wraps the return value in a new Promise anyway because the function is async, so it makes no different.